Corona-Warn-App/src/main/java/de/rki/coronawarnapp/familytest/core/repository/FamilyTestRepository.kt

```diff
@@ -68,15 +68,15 @@ class FamilyTestRepository @Inject constructor(
         storage.update(updated)
     }
 
-    suspend fun moveTestToRecycleBin(
+    suspend fun recycleTest(
```

Comment:
I changed that because recycle has a different meaning, how we use it is very confusing...

Reply:
ok ,renamed it in both repos for consistency